README.md

```diff
@@ -47,23 +47,42 @@ Testing this is complicated. Here is one approach:
 
 Production releases to S3 are based on the contents of the /dist folder and are built automatically by GitHub Actions for each branch pushed to GitHub and each merge into production.
 
-Merges into production are deployed to https://activity-player.concord.org/ (NOTE: production branch has not yet been created and deployment still needs to be configured).
+The production branch is deployed to https://activity-player.concord.org/
 
 Other branches are deployed to https://activity-player.concord.org/branch/<name>.
 
 To deploy a production release:
 
-1. Increment version number in package.json
-2. Create new entry in CHANGELOG.md
-3. Run `git log --pretty=oneline --reverse <last release tag>...HEAD | grep '#' | grep -v Merge` and add contents (after edits if needed to CHANGELOG.md)
-4. Run `npm run build`
-5. Copy asset size markdown table from previous release and change sizes to match new sizes in `dist`
-6. Create `release-<version>` branch and commit changes, push to GitHub, create PR and merge
-7. Checkout master and pull
-8. Checkout production
-9. Run `git merge master --no-ff`
-10. Push production to GitHub
-11. Use https://github.com/concord-consortium/activity-player/releases to create a new release tag
+1. Copy CHANGES-template.md to CHANGES.md, and add a list of PT stories related to the release into temporary CHANGES.md
+    - Run `git log --reverse <last release tag>...HEAD | grep '#'` to see a list of PR merges and stories that include PT ids in their message
+    - In a PT workspace that includes Orange and Teal boards, search for the `label:"activity-player-<new version>" includedone:true`. You can select all, and export as CSV. Then copy the id and title columns.
+2. Compute asset sizes.
+    1. Run `npm run build`
+    2. Look at file sizes with `ls dist/assets`
+    3. Add file sizes to CHANGES.md
+    4. Look at previous version file sizes from previous version in GitHub and compute the percent change `(new - prev) / prev * 100`
+3. Update package, commit, and tag
+    - **Mac or Linux**:
+        - Run `npm version -m "$(< CHANGES.md)" [new-version-string]`
+        - This updates the version in package.json and package-lock.json and creates a commit with the comment from CHANGES.md, and creates a tag with the name `v[new-version-string]` that has a description based on CHANGES.md.
+    - **Windows**: the command above that injects `CHANGES.md` as the message won't work in the standard windows command application.
+        - git-bash: same as above
+        - PowerShell: `npm version -m "(type CHANGES.md)" [new-version-string]` might work, I haven't tried it though.
+        - Do the steps manually and use a git client so you can paste in the multi line message
+            1. update package.json with the new version
+            2. run `npm install` to update package-lock.json
```

Comment:
You can accomplish 1 & 2 without tagging/committing with:
```
`npm version --no-git-tag-version [patch|minor|major]`
```
That's what I've generally done on several of my projects.

Reply:
Good point. I used `[new-version-string]` instead of `[patch|minor|major]` since we are typically trying to match this up with labels in PT.